components/src/core/components/TableSidebar/table-left-panel.scss

```diff
@@ -107,3 +137,17 @@
     }
   }
 }
+.table-header-action-secondary-btn{
+  display: inline-flex;
+  background-color: #f0fae4;
+  :deep(.oxd-svg-icon) {
+    height: 1.3rem;
+    width: 1.3rem;
+  }
+  :deep(.cls-1) {
+    fill: $oxd-button-color--secondary;
+  }
+  &.collapsed-btn {
+    margin-right: 0.5rem !important;
```

Comment:
Check why these are needed with !important. If it is the page specific override, please remove it